### PR TITLE
Magiclysm name to object

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
```SUMMARY: none```

#### Purpose of change
Contributes to #36110

#### Describe the solution
Change name to object
The names in data/mods/Magiclysm/items/ethereal_items.json have a weird plural, maybe if @KorGgenT glances over this pr he can comment on it, otherwise if this is merged it will just be the same as now.

#### Describe alternatives you've considered
Not changing

#### Testing
image
image
image

### CODE REVIEW

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 1]
```diff
@@ -2,8 +2,7 @@
   {
     "id": "stonefist",
     "type": "GENERIC",
-    "name": "pair of magical armored stone gauntlets",
-    "name_plural": "pairs of armored gauntlets",
+    "name": { "str": "pair of magical armored stone gauntlets", "str_pl": "pairs of armored gauntlets" },
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/ethereal_items.json]:
```suggestion
    "name": { "str": "pair of magical armored stone gauntlets", "str_pl": "pairs of magical armored stone gauntlets" },
```

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 2]
```diff
@@ -42,8 +41,7 @@
     "id": "armor_stonefist",
     "type": "ARMOR",
     "category": "armor",
-    "name": "pair of magical armored stone gauntlets",
-    "name_plural": "pairs of armored gauntlets",
+    "name": { "str": "pair of magical armored stone gauntlets", "str_pl": "pairs of armored gauntlets" },
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/ethereal_items.json]:
```suggestion
    "name": { "str": "pair of magical armored stone gauntlets", "str_pl": "pairs of magical armored stone gauntlets" },
```

**Comment by snipercup** [comment]
So you are okay with
```
"name": { "str": "pair of slick icy coatings on your feet", "str_pl": "slick icy coatings" },
```
?

**Comment by KorGgenT** [comment]
i thought i made a code suggestion for those, maybe it didn't take.

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 3]
```diff
@@ -117,8 +115,7 @@
   {
     "id": "ice_gliders",
     "type": "ARMOR",
-    "name": "pair of slick icy coatings on your feet",
-    "name_plural": "slick icy coatings",
+    "name": { "str": "pair of slick icy coatings on your feet", "str_pl": "slick icy coatings" },
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/ethereal_items.json]:
```suggestion
    "name": { "str": "slick icy coatings", "str_pl": "slick icy coatings" },
```

**Comment by snipercup** [comment]
It's ready now.

**Pull request merged**